source/docs/software/can-devices/third-party-devices.rst

```diff
@@ -19,56 +19,56 @@ CTR Electronics (CTRE) offers several CAN peripherals with external libraries. G
 
 ### CTRE Motor Controllers
 
-- **Talon FX (with Falcon 500 Motor)**
+- **Talon FX (with Falcon 500 Motor, Kraken x60 motor, and Kraken x44)**
 
-    - API Documentation (v5: [Java](https://api.ctr-electronics.com/phoenix/release/java/com/ctre/phoenix/motorcontrol/can/WPI_TalonFX.html), [C++](https://api.ctr-electronics.com/phoenix/release/cpp/classctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_w_p_i___talon_f_x.html) | Pro: [Java](https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html), [C++](https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html))
-    - [Hardware User's Manual](https://store.ctr-electronics.com/content/user-manual/Falcon%20500%20User%20Guide.pdf)
-    - Software Documentation ([v5](https://v5.docs.ctr-electronics.com/en/stable/ch13_MC.html), [Pro](https://pro.docs.ctr-electronics.com/en/stable/docs/api-reference/examples/quickstart.html))
+    - API Documentation (v6: [Java](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/hardware/TalonFX.html), [C++](https://api.ctr-electronics.com/phoenix/release/cpp/classctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_talon_f_x.html) | Pro: [Java](https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html), [C++](https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html))
```

Comment:
the pro API links should be removed. They were superseded by v6.

Reply:
Should be fixed now.